### DISCUSSION TITLE
Block src attributes also (on iframe, video, img html tags at least)

### DISCUSSION THREAD

**Discussion by l4nj1n9** [proginosko/LeechBlockNG#402] [Ideas]
it is possible to bypass the blocking by using a site that proxy the content of blocked sites. For example, Bing allows to access videos from Youtube even when Youtube itself is blocked by LeechBlockNG.

I think the solution should be to block requests, not only tabs.

BTW: **very good job**, certainly the **best tool I know** to fight procrastination.

**Comment by brrrom** [comment]
Yes! Blocking by MIME type in the same way keywords work would be a game changer!

There are a couple tools I've found that do this, for instance you can block application/vnd.yt-ump to block youtube videos from being playable. Or images/* or images/jpeg to block all images or images by type. 

It's super super useful and lets you curate your browsing experience so that most pages load much faster and don't show you distracting junk.

If this functionality was added into leechblock (the best blocker extension) it would be incredible. I love the simplicity of how the blocking works. If MIME types worked like keywords do so that you could apply to all sites with the simple * or get granular on a per domain basis that would be amazing. I love the flexibility to go allow list or block list mode or both with the extension.

The other tool I've seen do this will add a transparent tiny image in place of visual resources that it blocks on a page. So instead of seeing a broken link icon, you get a nice transparent empty space where the visual was, it's very nice!
